### PR TITLE
Updates to readmes and git templates

### PR DESCRIPTION
**Pull request by rmccar**
**Updates to readmes and git templates**

- Removing individual repo git templates
- Updating readmes with contents lists

### CODE REVIEW

**Comment by samiwel** [comment]
[Updates to project documentation in the monorepo.](https://trello.com/c/vQ1P19qL/747-updates-to-project-documentation-in-the-monorepo)

**Comment by jonnyshaw89** [comment]
@rmccar you need to resolve the conflicts as `eq-author-graphql-schema` has now gone.

File: eq-publisher/yarn.lock [hunk 1]
```diff
@@ -5,14 +5,12 @@
 "@babel/code-frame@^7.0.0", "@babel/code-frame@^7.0.0-beta.35":
   version "7.0.0"
   resolved "https://registry.yarnpkg.com/@babel/code-frame/-/code-frame-7.0.0.tgz#06e2ab19bdb535385559aabb5ba59729482800f8"
-  integrity sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==
```
Comment (jonnyshaw89) [inline eq-publisher/yarn.lock]: Why have all the hashes been removed from the lock?

Comment (rmccar) [reply eq-publisher/yarn.lock]: This usually happens when I do a yarn install and I dont have the latest version of yarn. Ive reverted this now

**Comment by jonnyshaw89** [comment]
@rmccar You should squash this to 1 commit before we merge, to keep the commit history clean

**Pull request merged**